code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1546,6 +1546,13 @@
 	M.update_transform()
 	..()
 
+/datum/reagent/plastic_polymers
+	name = "plastic polymers"
+	id = "plastic_polymers"
+	description = "the petroleum based components of plastic, contact a coder if you can actually see this" 
```

Comment:
But you can't because the reagent is instantly consumed when it makes the plastic and is therefore inaccessible. If you can see it and have it stablised, its a bug. (unless im wrong)